target/i386/hax-all.c

```diff
@@ -571,6 +622,8 @@ static int hax_vcpu_hax_exec(CPUArchState *env)
         cpu_exec_end(cpu);
         qemu_mutex_lock_iothread();
 
+        hax_update_orbital_cpu_procs(cpu);
```

Comment:
This might have some important performance impact, so we should call it conditionally, only if we *need* this information. This could be solved by adding this macro:
```c
#define DEBUG_CPU_UI 0
```

And changing these lines to:
```suggestion
        if (DEBUG_CPU_UI) {
            hax_update_orbital_cpu_procs(cpu);
        }
```

I'd keep the default value of the macro as zero, and only change it to one if necessary. In the future, it could be changed at runtime.